lst_scripts/reorganize_dl1hiperta_to_dl1lstchain.py

```diff
@@ -197,11 +189,29 @@ def stack_by_telid(dl1_pointer):
             respective path
     """
 
-    tabs = [Table(tel.parameters.read()) for tel in dl1_pointer]
-    stacked_param = vstack(tabs)
+    tels_params = [Table(tel.parameters.read()) for tel in dl1_pointer]
+    try:
+        tel_ids = [tel['telId'][0] for tel in dl1_pointer]
+    except:
+        # if the tel_id column does not exist, we assign tel ids by simple iteration
+        tel_ids = [i+1 for i in range(len(tels_params))]
```

Comment:
Is this case for hipecta files ? Not other way of retrieving the `tel_id` inside the file ?

Reply:
For hipecta files I don't know 🤷‍♂️
Any idea ?

Reply:
Without a current and bug-corrected version of hipecta, I cannot make any suggestions. 
However, if it is not for the hipecta file, why the `try, except` ? The parameter's table must contain always a `Tel_*['telId]` argument right ? or is it just for a general case and to be sure that next stages won't crash ?

Reply:
Yes it is for hipecta files but is a generic solution.